### PR TITLE
Merge pull request #32 from Olexandr88/patch-1

### PR DESCRIPTION
**Pull request by Xaxxoo**
docs: readme

## Description

_Detailed description of proposed changes_

## Related Issue(s)

Closes #
## Checklist:

- [ ] Read the [contributing docs](../CONTRIBUTING.md) (if this is your first contribution)
- [ ] Verified this is not a duplicate of any existing pull request

- [ ] Performed a self-review of the code
- [ ] Rebased to the last commit of the target branch (or merged it into my branch)
- [ ] Linked the issues which this PR resolves
- [ ] Documented the changes in code (API docs will be generated automatically)

### CODE REVIEW

File: app/components/footer.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Image from "next/image";
 
 const Footer = () => {
   return (
-    <footer className="flex flex-col-reverse md:flex-row justify-between items-center w-full min-h-[100px] bg-[#000014] px-4 md:px-8 lg:px-20 py-6 md:py-0 gap-6 md:gap-4">
+    <footer className="relative flex flex-col-reverse md:flex-row justify-between items-center w-full min-h-[100px] bg-[#000014] px-4 md:px-8 lg:px-20 py-6 md:py-0 gap-6 md:gap-4 mt-[100px]">
       {/* Copyright section */}
```
Comment (Abeeujah) [inline app/components/footer.tsx:7]: Hi @Xaxxoo you shouldn't be modifying this component if I'm correct.

File: app/components/footer.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ const Footer = () => {
       </div>
 
       {/* Social icons section */}
-      <div className="flex items-center gap-6">
+      <div className="flex items-center gap-12">
         <Image
```
Comment (Abeeujah) [inline app/components/footer.tsx:16]: Neither should you be doing this.

File: app/components/swapper.tsx [hunk 3]
```diff
@@ -69,11 +70,28 @@ const Swapper = () => {
...
-  const handleTokenSwap = () => {
+
+const STABLE_TOKENS: Token[] = ['USDT', 'USDC'];
+
+const handleTokenSwap = () => {
+  // ensures at least one of the tokens is a stable coin
+  const isValidSwap = 
+    STABLE_TOKENS.includes(fromToken) || 
+    STABLE_TOKENS.includes(toToken);
+
```
Comment (Abeeujah) [inline app/components/swapper.tsx:81]: This is not the intended functionality you were to implement, you were supposed to make sure that when the swap button is clicked, an unstable token doesn't end up in the stable token's placeholder.

File: app/components/swapper.tsx [hunk 4]
```diff
@@ -113,13 +131,33 @@ const Swapper = () => {
...
+              <RefreshCcw 
+                size={24} 
+                color={
+                  STABLE_TOKENS.includes(fromToken) || STABLE_TOKENS.includes(toToken) 
+                    ? 'white' 
+                    : 'gray'
+                } 
+              />
+            </button>
+
```
Comment (Abeeujah) [inline app/components/swapper.tsx:155]: You wasn't supposed to change the position of the swap button.

**Review by Abeeujah — CHANGES_REQUESTED** [review]

**Comment by princeibs** [comment]
#64

**Pull request closed without merging**